catalog/app/containers/Bucket/requests/requestsUntyped.js

```diff
@@ -1,6 +1,7 @@
 import { join as pathJoin } from 'path'
 
 import * as dateFns from 'date-fns'
+import * as FP from 'fp-ts'
```

Comment:
why are these changes necessary?

Reply:
> Why I replaced R.pipe with FP.functions.pipe? Or why I removed `.then`? Or both?

I removed `return await func().then()` because it violated `return-await` rule. Eslint showed error.
I replaced `R.pipe` with `FP.functions.pipe` because I thought you want it.